ansible/README.md

```diff
@@ -97,6 +97,19 @@ Yes, in order to access the package repositories (we will perform either `yum in
 
 4) The Ansible playbook will download and install any dependencies needed to build OpenJDK
 
+## How do I run the playbook on a Windows Machine?
+
+As Ansible can't run on Windows, it has to run remotely. To test the playbook, a Vagrant VM can be used.
```

Comment:
Can run on windows so you have to run remotely?  Doesn't quite make sense to me....